### PR TITLE
Improve FIP-0014

### PR DESCRIPTION
**Pull request by arajasek**
Subsumes #81, with minor improvements from me (I can't push to the fork branch).

### CODE REVIEW

File: FIPS/fip-0014.md [hunk 1]
```diff
@@ -35,25 +35,40 @@ Security concerns from the community boils down to @nicola's following comment.
...
 
-## Incentives of attackers
+### Incentives of attackers
 Under the circumstances mentioned above, there wouldn't be any incentives for attackers to invest huge amount of time and money just to fake a discontinued proof.
 
-Additionally, the concern is addressed by @neogeweb3 at [here](https://github.com/filecoin-project/FIPs/pull/75#discussion_r586057806) and by @steven004 at [here](https://github.com/filecoin-project/FIPs/issues/56#issuecomment-785715308). In short, the general consensus is that it is not economically viable that attacker could develop custom hardware using tech that is a year+ from now.
+### Core dev meeting
+Many of the security considerations were also extensively discussed in [Filecoin Core Dev Meeting #13](https://github.com/filecoin-project/tpm/blob/master/Core%20Dev%20Meetings/Meeting%200013.md). Some of the key takeaways are...
+> We need to agree on final answers to:
```
Comment (jennijuju) [inline FIPS/fip-0014.md:52]:
```suggestion
> Core devs need to agree on final answers to:
```

File: FIPS/fip-0014.md [hunk 2]
```diff
@@ -35,25 +35,40 @@ Security concerns from the community boils down to @nicola's following comment.
...
+> - whether adopting this FIP is acceptable from a security perspective
+> - whether the community's overall preference is in fact for this FIP
+> 
+> Answers to both questions are tending towards "yes"
```
Comment (jennijuju) [inline FIPS/fip-0014.md:56]:
```suggestion
> The answer for the first question is tending towards "yes", and the answer to the second question should be found out via a community poll
```

File: FIPS/fip-0014.md [hunk 3]
```diff
@@ -35,25 +35,40 @@ Security concerns from the community boils down to @nicola's following comment.
...
+
+### Summary of security considerations
+The general consensus is that it is not economically viable that an attacker could develop custom hardware using tech that is a year+ from now. Some highlights of community comments are by @neogeweb3 at [here](https://github.com/filecoin-project/FIPs/pull/75#discussion_r586057806) and by @steven004 at [here](https://github.com/filecoin-project/FIPs/issues/56#issuecomment-785715308).
 
 ## Incentive Considerations
-Majority of the people who sealed sectors using V1 proof for 180 days will now most likely to extend them to 540 days.
+The majority of the people who sealed sectors using V1 proof for 180 days will now most likely to extend them to 540 days.
```
Comment (jennijuju) [inline FIPS/fip-0014.md:62]:
```suggestion
The majority of the people who sealed sectors using V1 proof for 180 days will now most likely to extend them up to 540 days. 
```

File: FIPS/fip-0014.md [hunk 4]
```diff
@@ -35,25 +35,40 @@ Security concerns from the community boils down to @nicola's following comment.
...
 
 ## Product Considerations
-This fip protects interests of miners who have contributed to filecoin's growth since mainnet launch.
+This FIP protects interests of miners who have contributed to Filecoin's growth since mainnet launch. 
+
+This FIP is both a both security/implementation decision and governance decision. The decision made in [v1.2.0](https://github.com/filecoin-project/lotus/releases/tag/v1.2.0) of not extending V1 proof sectors was not an obvious catch. It wasn't until @deltazxm tried to extend a V1 proof sector and got an [error message](https://github.com/filecoin-project/specs-actors/issues/1309) that people realized the decision had been made. What if we find a bug in V1.1 tomorrow? Do we silently let V1.1 proof expire too? We need to establish some kind of process to reach consensus on making decisions like this together with different kinds of stakeholders in the network especially when changing policy, network parameters or other equivalents.
```
Comment (jennijuju) [inline FIPS/fip-0014.md:71]:
```suggestion
This FIP is both a both security/implementation decision and governance decision. The decision made in [v1.2.0](https://github.com/filecoin-project/lotus/releases/tag/v1.2.0) of not allowing extending V1 proof sectors was not an obvious catch. It wasn't until @deltazxm tried to extend a V1 proof sector and got an [error message](https://github.com/filecoin-project/specs-actors/issues/1309) that people realized the decision had been made. What if we find a bug in V1.1 tomorrow? Do we silently let V1.1 proof expire too? We need to establish some kind of process to reach consensus on making decisions like this together with different kinds of stakeholders in the network especially when changing policy, network parameters or other equivalents.
```

**Review by Fatman13 — APPROVED** [review]
Looks great! Thank you!

File: FIPS/fip-0014.md (hunk 4 above)
Comment (arajasek) [reply FIPS/fip-0014.md:71]: good point!

**Review by jennijuju — APPROVED** [review]

**Review by deltazxm — COMMENTED** [review]
LGTM

**Pull request merged**